### DISCUSSION TITLE
Issue Regarding Security Onion Installation - Ubuntu

### DISCUSSION THREAD

**Discussion by SOS-H** [Security-Onion-Solutions/securityonion#6686] [General]
![Screenshot from 2021-12-23 15-05-42](https://user-images.githubusercontent.com/96569907/147226115-73aa6243-fd5e-4c84-91d6-1a9f58dcff01.png)
Dear all,
plz help me out of this!!
while i am Verify the downloaded ISO image using the signature file it shows the error which i attached
Thanks in advance.

**Comment by tomjones1977** [comment]
Are you in the right folder?  Working on a couple of assumptions here, Ubuntu browsers generally save files to the `~/Downloads` folder, so the command above can't find the file to verify against.
